.github/workflows/release.yaml

```diff
@@ -0,0 +1,38 @@
+name: Release zbchaos
+on:
+  workflow_dispatch:
+    inputs:
+      release:
+        description: 'Release and tag name'     
+        required: true
+        default: 'zbchaosv0.X.0'
```

Comment:
:wrench: 
```suggestion
        default: 'zbchaos v1.X.0'
```

Reply:
@oleschoenburg I think we cant do it because the tag will be created with it, but how about using a `-`.

Reply:
Ah okay yeah. Then we need to change this too I think: https://github.com/zeebe-io/zeebe-chaos/blob/main/.github/workflows/publishZbchaosImage.yml#L24